src/components/LoginForm.vue

```diff
@@ -283,7 +287,7 @@ const loginBtnText = computed((): string => {
     return ''
   }
 
-  return messages.login.loginText
+  return loginButtonText.value ? loginButtonText.value : messages.login.loginText
```

Comment:
I'm not sure you need to check if undefined since you're setting the fallback string in the `inject` call?